ext/nnstreamer/tensor_filter/tensor_filter_mxnet.cc

```diff
@@ -0,0 +1,547 @@
+ *                   outputname=argmax_channel \
+ *                   custom=input_rank=4,enable_tensorrt=false \
+ *                   accelerator=true:cpu,!npu,!gpu \
+ *               ! appsink",
+ *
+ * @note Special considerations on props:
+ *
+ *   outputname:
+ *     The output name should not be the name of output layer, but the name of
+ *     MXNet NDArray operator passed to the output. For example, the most common
+ *     name is "argmax_channel" for typical classification model. It also can be
+ *     "sigmoid" or etc.
+ *     If it is not meant to do any operations, "_copyto" operator may be used.
+ *     The output will be passed as-is.
+ *     The current limitation is that it only takes a unary operator, meaning it
+ *     does not take additional parameters.
+ *     The candidates of operators can be found here:
+ *      - https://github.com/apache/incubator-mxnet/blob/1.7.0/benchmark/opperf/nd_operations/unary_operators.py#L24-L28
+ *      - https://mxnet.apache.org/versions/1.7/api/python/docs/api/ndarray/op/index.html?highlight=argmax_channel#mxnet.ndarray.op.argmax_channel
+ *
+ *     If this prop is not configured appropriately, the application may get
+ *     unexpected output values, or even segfaults without warning messages.
+ *
+ *   custom:
+ *     Each entries are separated by ','
+ *     Each entries have property_key=value format.
+ *     There must be no speces.
+ *
+ *     Supported props:
+ *       input_rank: (mandatory)
+ *            Rank of each input tensors.
+ *            Each ranks are separeted by ':'.
+ *            The number of ranks must be the same as the number of input
+ *            tensors.
+ *       enable_tensorrt: (optional)
+ *            set true to enable NVIDIA TensorRT. GPU acceleration must be
+ *            enabled in order to use it.
+ *
+ *     Examples:
+ *       tensor_filter framework=mxnet model=model/Inception-BN.json
+ *                input=1:3:224:224
+ *                inputname=data
+ *                accelerator=true:!cpu,!npu,gpu
+ *                ...
+ *                custom=input_rank=4,enable_tensorrt=true
+ *
+ *        tensor_filter framework=mxnet model=model/Inception-BN.json
+ *                input=1:3:224:224,1
+ *                inputname=data1,data2
+ *                ...
+ *                custom=input_rank=4:1
+ */
+#include <nnstreamer_cppplugin_api_filter.hh>
+#include <nnstreamer_util.h>
+#include <stdexcept>
+#include <tensor_common.h>
+#include <unistd.h>
+
+#include <fcntl.h>
+#include <glib.h>
+#include <glib/gstdio.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include <mxnet-cpp/MxNetCpp.h>
+
+using nnstreamer::tensor_filter_subplugin;
+using namespace mxnet::cpp;
+
+namespace nnstreamer
+{
+namespace tensorfilter_mxnet
+{
+
+const static std::string kFileLocation = "/ext/nnstreamer/tensor_filter/tensor_filter_mxnet.cc";
+const static std::string kFileUrl
+    = "https://github.com/nnstreamer/nnstreamer/tree/main" + kFileLocation;
```

Comment:
Is it necessary? It looks possible to print good error messages without ```kFileUrl```

Reply:
Changed like
```cpp
const static std::string kFileLocation
    = "<nnstreamer_repo_home>/ext/nnstreamer/tensor_filter/tensor_filter_mxnet.cc";
```
And replace usage of kFileUrl to use kFileLocation.

Reply:
@minty99 
If you want to specify the file where the errors occur, you can find more elegant ways (e.g., built-in macros supported by GCC or CLANG) than using the hard-coded path.

Reply:
@wooksong It seems what you mentioned is `__FILE__` macro, so I changed `kFileLocation` to `__FILE__`.